.travis.yml

```diff
@@ -37,6 +37,9 @@ jobs:
       install:
         - pip install -U pip certifi
         - pip install $PIP_DEPENDENCIES
+        # Since there is no version of stdatamodels on PyPI, install from
+        # Github master until it is released to PyPI
+        - pip install git+https://github.com/spacetelescope/stdatamodels
```

Comment:
Is this necessary?  I would have thought your setup.cfg requirement would be enough to get it installed from source.

Reply:
Unfortunately it is, as for this job the pip install script is

```
pip install -r requirements-min.txt .[test]
```

where `requirements-min.txt` is generated by the `minimum_deps` script, which takes the `install_requires` and finds the minimum version available on pypi and uses that.  For `stdatamodels`, the minimum and only version is 0.0.0, which doesn't work.  So we need to install the latest version for that package.  Once we deliver a proper version of `stdatamodels` to pypi, we can remove this.